src/lib/gundb/UserStorageClass.js

```diff
@@ -1256,7 +1256,9 @@ export class UserStorage {
       feed
         .filter(
           feedItem =>
-            feedItem.data && ['deleted', 'cancelled'].includes(feedItem.status || feedItem.otplStatus) === false
+            feedItem &&
```

Comment:
@sirpy I’ve asked @Yaroslav Fedyshyn to add this hot fix. because sometimes I get error cannot read data from undefined in this function (and I don’t see all feed. it is empty). with this fix it works well